packages/fast-components-react-msft/src/button/button.spec.tsx

```diff
@@ -133,4 +152,36 @@ describe("button", (): void => {
 
         expect(button.prop("className")).toEqual(undefined);
     });
+
+    /* tslint:disable-next-line */
+    test("should add a child element with the 'before' prop into the before slot", () => {
+        const props: IButtonHandledProps = {
+            appearance: ButtonAppearance.lightweight,
+            href: "#",
+            children: ["foo", beforeSlotExample]
+        };
+
+        const rendered: any = mount(
+            <Component {...props} />
+        );
+
+        expect(rendered.instance().props.children[1].props.slot).toBe("before");
+        expect(rendered.find("div.slotBefore").length).toBe(1);
+    });
+
+    /* tslint:disable-next-line */
+    test("should add an 'after' element when a child is passed with the after slot", () => {
```

Comment:
can you also change this to match the above test string, something like "should add a child element with the slot prop set to 'after' into the after slot location" (also add similar wording above, my previous comment called it the 'before' prop but its the slot prop)